<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">SVG Cleaner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons/icons.qrc">
    <normaloff>:/svgcleaner.svgz</normaloff>:/svgcleaner.svgz</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="ItemsWidget" name="itemsWidget" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QScrollBar" name="itemsScroll">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string notr="true">ToolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWizard"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionCompareView"/>
   <addaction name="actionInfo"/>
  </widget>
  <action name="actionWizard">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/wizard.svgz</normaloff>:/wizard.svgz</iconset>
   </property>
   <property name="text">
    <string notr="true">wizard</string>
   </property>
   <property name="iconText">
    <string notr="true">wizard</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/start.svgz</normaloff>:/start.svgz</iconset>
   </property>
   <property name="text">
    <string notr="true">start</string>
   </property>
   <property name="iconText">
    <string notr="true">start</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/stop.svgz</normaloff>:/stop.svgz</iconset>
   </property>
   <property name="text">
    <string notr="true">stop</string>
   </property>
   <property name="iconText">
    <string notr="true">stop</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/information.svgz</normaloff>:/information.svgz</iconset>
   </property>
   <property name="text">
    <string notr="true">info</string>
   </property>
   <property name="iconText">
    <string notr="true">info</string>
   </property>
   <property name="toolTip">
    <string>Information</string>
   </property>
  </action>
  <action name="actionCompareView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/thumbs-on.svgz</normaloff>:/thumbs-on.svgz</iconset>
   </property>
   <property name="text">
    <string notr="true">compareView</string>
   </property>
   <property name="iconText">
    <string notr="true">compareView</string>
   </property>
   <property name="toolTip">
    <string>Compare view</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/pause.svgz</normaloff>:/pause.svgz</iconset>
   </property>
   <property name="text">
    <string notr="true">pause</string>
   </property>
   <property name="iconText">
    <string notr="true">pause</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemsWidget</class>
   <extends>QWidget</extends>
   <header>itemswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
